Snippets/Snippets_6/UpgradeGuides/5to6/CustomErrorHandlingBehaviour.cs

```diff
@@ -0,0 +1,56 @@
+namespace Snippets6.UpgradeGuides._5to6
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.Pipeline;
+
+    #region 5to6-customerrorhandlingbehaviour
+    class CustomErrorHandlingBehaviour : Behavior<ITransportReceiveContext>
+    {
+        public override async Task Invoke(ITransportReceiveContext context, Func<Task> next)
+        {
+            try
+            {
+                await next().ConfigureAwait(false);
```

Comment:
@SimonCropp - thoughts?

>@SimonCropp - I am not sure, that it improves readability by removing ConfigureAwait(false). 
It's just one line of code and it follows best practice. I actually see no harm in leaving it the way it currently is. Anyone wanting to add stuff to the pipeline will be looking at the samples and I think it serves best to point them to the right way of doing things, in this case, by using ConfigureAwait(false) - thoughts?